### DISCUSSION TITLE
Support for non-rectangular regions

### DISCUSSION THREAD

**Discussion by HadrienGardeur** [readium/guided-navigation#5] [General]
In 2023 during a number of calls organized to kickstart these efforts, we quickly identified that support for rectangular regions in an image is a must have. To support them, we can directly reference Media Fragments URI 1.0: https://www.w3.org/TR/media-frags/#naming-space

In addition to rectangles, EPUB Region-Based Navigation 1.0 also introduces two other shapes:

- circular regions: https://idpf.org/epub/renditions/region-nav/#sec-3.5.1.2
- polygonal regions: https://idpf.org/epub/renditions/region-nav/#sec-3.5.1.3

We already had a consensus on polygonal regions which feel necessary for borderless panels and to identify regions in a panel (a character for example), but we never really discussed circular regions that much.

**Comment by llemeurfr** [comment]
I would simply disregard circular regions because they correspond to a narrow use case.
Rectangular regions are a basic way to isolate a screen section.
Polygonal sections are a complex but sophisticated way to customize a shape. 
The navigation editor does not need something else, IMHO.
